sites/partners/src/components/applications/ApplicationsColDefs.ts

```diff
@@ -562,7 +562,7 @@ export function getColDefs(maxHouseholdSize: number, countyCode: string) {
         width: 125,
         minWidth: 100,
         valueFormatter: ({ value }) => {
-          if (!value) return ""
+          if (value.length < householdIndex) return ""
```

Comment:
Previously, we were showing `n/a` just in these columns for otherwise empty household members